src/modules/rlm_ldap/user.c

```diff
@@ -230,11 +230,11 @@ void rlm_ldap_check_reply(module_ctx_t const *mctx, request_t *request, fr_ldap_
 	*/
 	if (!inst->expect_password || !RDEBUG_ENABLED2) return;
 
-	if (!fr_pair_find_by_da(&request->control_pairs, NULL, attr_cleartext_password) &&
-	    !fr_pair_find_by_da(&request->control_pairs, NULL, attr_nt_password) &&
+	if (!fr_pair_find_by_da_nested(&request->control_pairs, NULL, attr_cleartext_password) &&
```

Comment:
This could arguably be changed to look for `parent` as the `Password` attribute.  If that isn't found, set `parent = request->control_pairs`

and then just do `fr_pair_find_by_da(parent, NULL, attr_cleartext_password)` etc.  That avoids some extra work.